### PR TITLE
Context homework

### PR DESCRIPTION
**Pull request by Rehaeel**

### CODE REVIEW

File: 3.JavaScript-functions_context_this/script.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    let firstName = first,
+        lastName = last,
+        fullName = `${firstName} ${lastName}`;
+
+    return {
+        set firstName(name) {
+            firstName = name;
+            fullName = `${firstName} ${lastName}`;
+        },
+        get firstName() {
```
Comment (jbbch) [inline 3.JavaScript-functions_context_this/script.js:13]: you don't have to specify getter if it's behaviour is default. same with setter

Comment (Rehaeel) [reply 3.JavaScript-functions_context_this/script.js:13]: Isn't it more readable though?

Comment (Rehaeel) [reply 3.JavaScript-functions_context_this/script.js:13]: > 

When I delete `set` and `get` , methods stop working. I'm not sure what did you mean.

Comment (Rehaeel) [reply 3.JavaScript-functions_context_this/script.js:13]: When I deleted `get` and `set` , the methods stopped working. I'm not sure what did you mean then.

Comment (jbbch) [reply 3.JavaScript-functions_context_this/script.js:13]: My bad, you actually have to use both getter and setter :)

**Pull request merged**